FeedAPIChallenge/Feed API/FeedItemsMapper.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  FeedItemsMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Valeria Chub on 10.09.2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedItemsMapper {
+	private static let STATUS_OK_200 = 200
+
+	internal static func map(data: Data, response: HTTPURLResponse) -> FeedLoader.Result {
+		guard
+			response.statusCode == STATUS_OK_200,
+			let root = try? JSONDecoder().decode(Root.self, from: data) else {
+			return .failure(RemoteFeedLoader.Error.invalidData)
+		}
+
+		return .success(root.feedImages)
+	}
+}
+
+extension FeedItemsMapper {
+	private struct Root: Decodable {
+		private let items: [Item]?
+
+		fileprivate var feedImages: [FeedImage] {
+			return items?.map { $0.feedImage } ?? []
+		}
+	}
+
+	private struct Item: Decodable {
+		private let id: UUID
+		private let description: String?
+		private let location: String?
+		private let url: URL
+
+		private enum CodingKeys: String, CodingKey {
+			case id = "image_id"
+			case description = "image_desc"
+			case location = "image_loc"
+			case url = "image_url"
+		}
+
+		fileprivate var feedImage: FeedImage {
```

Comment:
Same here as for `Root.feedImages`
```suggestion
		var feedImage: FeedImage {
```